contrib/storage-mongo/src/main/java/org/apache/drill/exec/store/mongo/MongoGroupScan.java

```diff
@@ -503,7 +505,9 @@ public ScanStats getScanStats() {
       long numDocs = collection.count();
       float approxDiskCost = 0;
       if (numDocs != 0) {
-        String json = collection.find().first().toJson();
+        final DocumentCodec codec =
```

Comment:
Please add a comment to describe the need for the codec along with the example from the bug.